src/common/home/Home.jsx

```diff
@@ -1,4 +1,4 @@
-import { useState, useEffect, useContext } from "react";
+import { useState, useEffect, useContext} from "react";
```

Comment:
Thanks for correcting it. COuld you please retain the space before the closing } as it was before?

@Anirban-1490

Reply:
> Thanks for correcting it. COuld you please retain the space before the closing } as it was before?
> 
> @Anirban-1490

ok